cmd/vclusterctl/cmd/delete.go

```diff
@@ -107,7 +117,13 @@ func (cmd *DeleteCmd) Run(cobraCmd *cobra.Command, args []string) error {
 	cmd.log.Infof("Delete vcluster %s...", args[0])
 	err = helm.NewClient(cmd.rawConfig, cmd.log, helmBinaryPath).Delete(args[0], cmd.Namespace)
 	if err != nil {
-		return err
+		if cmd.IgnoreNotFound {
+			if strings.Contains(err.Error(), "was not found in namespace") {
+				cmd.log.Donef("vcluster %s not found in namespace %s, ignoring since --ignore-not-found flag is set", args[0], cmd.Namespace)
+			}
```

Comment:
Is a nested if block required here, can't both conditions be clubbed together (like if cond. A && cond. B)

Reply:
`strings.Contains(err.Error(), "was not found in namespace")`
Do we have any specific keyword or phrase in the error statement to match? If not, then let's convert this into an error const. Like const FlagNotFound = "not found in namespace".

Reply:
Sure, I will club the conditions together. 
As for the error messages, we have two error messages:
1. `couldn't find vcluster`
2. `was not found in namespace` 
That we need to take care of and ignore them, instead of an exit code

Reply:
@ApsTomar I have clubbed the conditions and added the messages to const variables.